lib/model/frames/form.js

```diff
@@ -174,7 +178,7 @@ Form.ExtendedVersion = Frame.define('excelContentType', readable);
 ////////////////////////////////////////////////////////////////////////////////
 // FORM DEF
 
-Form.Def = Frame.define(
+Form.Def = class FormDef extends Frame.define(
```

Comment:
```suggestion
Form.Def = class extends Frame.define(
```

Not 100% sure, but maybe specifying a class name here isn't idiomatic for the codebase? E.g., I see `Form.Field = class extends Frame.define(` below, where the class doesn't have an explicit name.

Reply:
You're correct that class names are not otherwise included for sub-classes:

```
$ git grep '= class.*extends Frame\.define'
lib/model/frames.js:Form.Attachment = class extends Frame.define(
lib/model/frames.js:Project.Extended = class extends Frame.define(
lib/model/frames.js:Submission.Attachment = class extends Frame.define(
lib/model/frames/dataset.js:Dataset.Extended = class extends Frame.define(
lib/model/frames/entity.js:Entity.Def.Source = class extends Frame.define(
lib/model/frames/form.js:Form.Extended = class extends Frame.define(
lib/model/frames/form.js:Form.Field = class extends Frame.define(
```

I will follow this idiom, but suggest changing it elsewhere.

Reply:
> Including a name for classes is likely helpful in stacktraces.

> I will follow this idiom, but suggest changing it elsewhere.

I'd be game for us to change the idiom and give all classes a name (probably in a follow-up PR). That makes sense that that would help in stacktraces.